### PR TITLE
More info on plugins, plus note on new core function

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: content/dev/plugins/encapsulated_plugins/converting.md [hunk 1]
```diff
@@ -67,3 +67,27 @@ In our example, these will be placed in
...
+Some options for overcoming this are: 
+
+- Move the logic to a page specific init file. 
+
+- Make the variables into defined constants (which by definition have global scope). 
+
+### 6. Remove old files and update documentation 
+
+Please *do not* preserve top level folders like `admin` and `includes` with the same files you have created in `zc_plugins/YOURPLUGIN/VERSION/`.  Instead, update the README file for your plugin with guidance like: 
+
```
Comment (drbyte) [inline content/dev/plugins/encapsulated_plugins/converting.md:90]: > do not preserve top level folders like `admin` and `includes`

"do not preserve these folders" seems confusing, as one might interpret it as "under zc_plugins don't create any directories named "admin" or "includes". But that's contradictory to what does need to be done.

Did you mean "don't just copy all your old-style files into zc_plugins and call it a day; instead, review every file and ensure its relevance, suitability, and proper location and function, rewriting whatever parts are necessary to conform to the new requirements, and taking the time to rewrite things that could be modernized, and removing things that are just carryovers of old ways of doing things"? or something to that effect?

File: content/user/upgrading/release_specific_upgrade_considerations.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ This document lists things you may wish to take into account as you upgrade.  Th
 - changes to the database schema or contents which deserve special notice
 - changes to basic template operation 
 
+### Zen Cart 1.5.8
+
+- The function `zen_cfg_read_only` has been added to the core.  This means any plugin authors who use this function should wrap it in `if (!function_exists('zen_cfg_read_only'))`. 
+
```
Comment (drbyte) [inline content/user/upgrading/release_specific_upgrade_considerations.md:16]: IIRC there's only one plugin that ever did that, right?

Comment (scottcwilson) [reply content/user/upgrading/release_specific_upgrade_considerations.md:16]: No there's a few in addition to the sitemap. 

Google_mapinator
Multi-Language-Country-Names
dangling_carrot

And there may be others that are not in the Plugins area.   (Installed modifications, commercial mods, etc.)

File: content/dev/plugins/encapsulated_plugins/converting.md (hunk 1 above)
Comment (scottcwilson) [reply content/dev/plugins/encapsulated_plugins/converting.md:90]: Not exactly ... I meant "don't replicate them at the top level as well."   Let me rework some of this language.

**Pull request merged**